### PR TITLE
Bootstrap cli commands

### PR DESCRIPTION
**Pull request by djhi**
- [x] Bootstrap init
- [x] Bootstrap add todo
- [x] Tests

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,12 +1,16 @@
 {
-    "extends": ["eslint:recommended", "prettier"],
+    "extends": [
+        "eslint:recommended",
+        "prettier"
+    ],
     "plugins": ["prettier"],
     "parserOptions": {
-        "ecmaVersion": 2017,
+        "ecmaVersion": 2018,
```
Comment (jdemangeon) [inline .eslintrc:8]: :sunglasses:

File: packages/cli/package.json [hunk 2]
```diff
@@ -7,10 +7,34 @@
...
   },
   "lint-staged": {
-    "src/**/*.js": ["yarn run lint", "git add"]
+    "src/**/*.js": [
+      "yarn run lint",
+      "git add"
+    ]
+  },
+  "dependencies": {
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.3",
```
Comment (jdemangeon) [inline packages/cli/package.json]: Not in devDependencies?

File: packages/cli/src/api/update-project.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import debug from 'debug';
+
+const updateProject = (projectName, dependencies) =>
+    new Promise(resolve => {
+        debug('api.update-project')(
+            `Saving ${projectName} with ${dependencies.length} dependencies`
+        );
+
+        setTimeout(
+            () => resolve(`https://eydia.marmelab.com/project/${projectName}`),
```
Comment (jdemangeon) [inline packages/cli/src/api/update-project.js:10]: We need a config for that

File: packages/cli/src/commands/init/index.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    });
+};
+
+const handleSteps = (
+    dispatch,
+    { step, projectName, dependencies },
+    onExit,
+    projectRoot
+) => () => {
+    if (step === 'welcome') {
```
Comment (jdemangeon) [inline packages/cli/src/commands/init/index.js:80]: It's the best place to use a switch/case :p

File: packages/cli/package.json (hunk 2 above)
Comment (djhi) [reply packages/cli/package.json]: good catch

File: packages/cli/src/api/update-project.js (hunk 3 above)
Comment (djhi) [reply packages/cli/src/api/update-project.js:10]: Yes but it's too soon

File: packages/cli/src/commands/init/index.js (hunk 4 above)
Comment (jdemangeon) [reply packages/cli/src/commands/init/index.js:80]: Moreover, you can maybe use constants for "step" names, they're used at many places

File: packages/cli/src/index.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+            )}
+        </>
+    );
+};
+
+render(
+    <AppContext.Consumer>
+        {({ exit }) => <Root onExit={exit} />}
+    </AppContext.Consumer>
+    // {
```
Comment (jdemangeon) [inline packages/cli/src/index.js:42]: debug ?

**Pull request merged**